tests/console-backend-service/internal/domain/cms/clusterdocstopic_test.go

```diff
@@ -255,17 +261,17 @@ func fixClusterDocsTopicMeta(name, order string) metav1.ObjectMeta {
 	}
 }
 
-func fixCommonClusterDocsTopicSpec() v1alpha1.CommonDocsTopicSpec {
+func fixCommonClusterDocsTopicSpec(host string) v1alpha1.CommonDocsTopicSpec {
 	return v1alpha1.CommonDocsTopicSpec{
 		DisplayName: fixture.DocsTopicDisplayName,
 		Description: fixture.DocsTopicDescription,
 		Sources: []v1alpha1.Source{
 			{
-				Type:       "openapi",
-				Name:       "openapi",
+				Type:       "markdown",
+				Name:       "markdown",
 				Parameters: &runtime.RawExtension{Raw: []byte(`{"json":"true","complex":{"data":"true"}}`)},
 				Mode:       v1alpha1.DocsTopicSingle,
-				URL:        "https://petstore.swagger.io/v2/swagger.json",
+				URL:        fmt.Sprintf("http://%s/README.md", host),
```

Comment:
I think about making for that function. What do you think about that? If you agree with me, please also change inline code in `service-catalog` tests to new function.

Reply:
Created in `mockice` package.